src/pc/audio/audio_sdl.c

```
@@ -4,51 +4,178 @@
+    if (sMicrophoneDev == 0) return;
+    sCurrentlyCapturing = false;
+    SDL_PauseAudioDevice(sMicrophoneDev, 1);
+}
+
+static char** audio_sdl_list_devices(int* count, unsigned int* current, int type) {
+    int num = SDL_GetNumAudioDevices(type);
+    if (count) *count = num;
+
+    char** strings = malloc(num * sizeof(const char*));
+    for (int i = 0; i < num; i++) {
+        strings[i] = strdup(SDL_GetAudioDeviceName(i, type));
```

Cppcheck (2.20.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)